### DISCUSSION TITLE
Github Copilot Student

### DISCUSSION THREAD

**Discussion by gautier337** [community/community#30048] [Copilot Conversations] [answered]
Hello,

I searched if copilot was free for students and i found then yes it is.

When i go into settings -> github copilot there is write: 'Your free access to GitHub Copilot is ending on August 22, 2022 at 9am PDT.'

I am a verified student do you know if there is a mistake or something ? 

Thank you for answer,
Have a good day

**Comment by Hugo-code-dev** [comment]
> Hello,
> 
> I searched if copilot was free for students and i found then yes it is.
> 
> When i go into settings -> github copilot there is write: 'Your free access to GitHub Copilot is ending on August 22, 2022 at 9am PDT.'
> 
> I am a verified student do you know if there is a mistake or something ?
> 
> Thank you for answer, Have a good day

I can relate the same thing

**Reply by Rufluenza** [reply]
I have the same issue

**Comment by veverkap** [comment]
I can confirm that you both will continue having access after August 22nd.  There is a bug (that we are fixing) for some users that shows this incorrect information. 

I'm sorry for the inconvenience and will let you know when it is fixed.

**Reply by Hugo-code-dev** [reply]
Perfect thank you for your answer

**Reply by jimeh** [reply]
I assume the bug affects people with free access due to open source contributions as well? Cause I have the same warning on my account, and I've also received an email about it too :)

**Reply by veverkap** [reply]
Yes, essentially anyone who was a part of the Technical Preview and has gotten access a different way since them could be impacted.

**Reply by Rufluenza** [reply]
Yeah i got the technical preview access and signed up for a 60 days then i got the student pack and it worked for a few days however it not says that my subscription has ended and that i am eligible for a free subscription but it says "your subscription has ended." if anyone knows how to fix i would love the help :D

**Comment by M1saka10010** [comment]
I have a same problem lol

**Comment by veverkap** [comment]
Would you all mind checking your https://github.com/settings/copilot page to see if this has been resolved.  I apologize again for the inconvenience.

@M1saka10010 @Hugo-code-dev @jimeh @gautier337

**Reply by jimeh** [reply]
It has been resolved indeed. The message is gone both from settings and the billings page. Thanks :)

**Reply by veverkap** [reply]
Thank you for your patience!

**Comment by alv18051** [comment]
I just enrolled as a student because of copilot but I have the same issue :(

**Reply by veverkap** [reply]
@alv18051 please visit https://github.com/github-copilot/free_signup and signup there.

**Reply by alv18051** [reply]
Thank you for your assistance:)

**Reply by vicanmendez** [reply]
> @alv18051 please visit https://github.com/github-copilot/free_signup and signup there.

Hi friend! I think I am having the same issue, but I can't find the solution, I had the trial version of Copilot, then I remembered that I am also a student in an online university program and I applied for GitHub Education. Now, I cant use Copilot and in my Settings, I just can see the message saying that my subscription has ended. I can't renew my subscription as a student and do not even see an option to purchase the service.

**Comment by vishnuvskvkl** [comment]
I'm also enrolled as student but now there is payment notification

**Reply by veverkap** [reply]
Please reach out to GitHub Support (https://support.github.com/) for assistance.

**Comment by mahdi-safdari** [comment]
@veverkap Hi, I have the same problem, even though I didn't use the free course at all, can you reactivate it?

**Reply by veverkap** [reply]
Please reach out to GitHub Support (https://support.github.com/) for assistance.

**Reply by mahdi-safdari** [reply]
How?

**Reply by veverkap** [reply]
https://support.github.com/contact <- create a support ticket.

**Reply by digfish** [reply]
When clicking in "new ticket" it gets redirected to https://support.github.com/request, the search page.

**Comment by digfish** [comment]
I'm a verified student and in VS code it says that my trial has expired. And in the billing page this is what shows up:
![image](https://user-images.githubusercontent.com/717142/185788895-b6e4f0e1-f6fd-4da4-aff6-dd5bda35f4f0.png)
It's not clear if my subscription as verified student stays active :(

**Reply by ZappaBoy** [reply]
Same here

**Reply by veverkap** [reply]
@digfish You will need to contact GitHub Support (https://support.github.com/) for assistance.

**Reply by veverkap** [reply]
@ZappaBoy please visit https://github.com/github-copilot/free_signup and signup there.  If you have any issues, please reach out to GitHub Support (https://support.github.com/) for assistance.

**Reply by digfish** [reply]
When searching for the question in https://support.github.com/) the search results point to this thread. This is circular redundancy.

**Reply by veverkap** [reply]
I meant to create a new support ticket with them.

**Reply by veverkap** [reply]
https://support.github.com/contact

**Reply by ZappaBoy** [reply]
> @ZappaBoy please visit https://github.com/github-copilot/free_signup and signup there. If you have any issues, please reach out to GitHub Support (https://support.github.com/) for assistance.

@veverkap Yeah, it works. Thanks 👍🏼.

**Comment by sabir-shah123** [comment]
I just applied for the Github student Pack with my university email and card , after submitting application i got email from github with info like welcome to GitHub global campus you are now part of this .
but when i opened setting of github for checking copilot 
it says 
"'Your free access to GitHub Copilot is ending on August 22, 2022 at 9am PDT. "

please let me know if i have to wait or i am doing something wrong

**Reply by veverkap** [reply]
You will need to contact GitHub Support https://support.github.com/contact and they can give you help.

**Comment by MSBARAKA** [comment]
Hello, I have the same problem and I want to send you my student certificate. Please Whom will I send it ??

**Reply by veverkap** [reply]
You will need to contact GitHub Support https://support.github.com/contact and they can give you help.

**Comment by taylor-lindores-reeves** [comment]
Same problem, I created a support ticket 1 week ago about this. I was charged today but I am a verified student (see my GitHub Pro charge is $0.00)..

<img width="484" alt="Screenshot 2022-08-22 at 16 27 03" src="https://user-images.githubusercontent.com/27443324/185959114-2271adca-5b20-40fd-b035-ec6c655c1f25.png">

**Comment by Davec876** [comment]
I have the same problem and I was billed for it even though I am verified student. Can you please help me fix this?

**Reply by veverkap** [reply]
You will need to contact https://support.github.com/contact for more assistance.  I can see that you are a free user but not much more than that.

**Comment by martinb-ai** [comment]
Mine says i have free use of it but my VS code does not let me use it and continues to complain to upgrade the plan. @veverkap any idea what I should do? Thanks!

**Reply by veverkap** [reply]
What notification are you getting in your editor?

**Reply by martinb-ai** [reply]
<img width="1132" alt="image" src="https://user-images.githubusercontent.com/105685594/186412628-6001a04a-f555-41a2-af3b-cbc465063ac1.png">

**Reply by veverkap** [reply]
Can you please upgrade your Copilot extension?

**Reply by martinb-ai** [reply]
Already did 😄 

<img width="658" alt="image" src="https://user-images.githubusercontent.com/105685594/186429059-167e9dfc-cda6-4c33-92ec-63172c943bcd.png">

**Reply by himanshu-sagar** [reply]
😃

**Reply by veverkap** [reply]
The newest version of the Copilot Extension is 1.42 based.

![image](https://user-images.githubusercontent.com/22348/186472515-b5d82a87-ae89-42ac-8164-6afd50f49a42.png)

Can you update your VS Code so that you can get that updated extension?

**Reply by martinb-ai** [reply]
fixed. thank you!

**Comment by himanshu-sagar** [comment]
Mine stopped working a few days ago. Is there any way to extend this?

**Reply by davecheney** [reply]
@himanshu-sagar as a technical preview user you received complementary access to copilot until August the 22nd. If you wish to continue to use Copilot, you can sign up for a subscription plan here , https://github.com/github-copilot/signup

**Comment by imsamuelcovalero** [comment]
Same issue here, Can you please help me fix this please?

**Comment by gautier337** [comment, marked as answer]
Finally, despite the fact that this was written, my subscription was **still extended**...
